src/components/Selfie/Intro.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import { h, Component } from 'preact'
+import PageTitle from '../PageTitle'
+import style from './style.css'
+import theme from '../Theme/style.css'
+import classNames from 'classnames'
+import Button from '../Button'
+import { localised } from '../../locales'
+import { trackComponent } from '../../Tracker'
+import withCrossDeviceWhenNoCamera from '../Capture/withCrossDeviceWhenNoCamera'
+import withCameraDetection from '../Capture/withCameraDetection'
+import { compose } from '~utils/func'
+
+const InstructionPure = ({instruction}) => (
+    <li key={instruction.key} className={style.introBullet}>
+      <span className={classNames(style.introIcon, style[`${instruction.key}Icon`])} />
+      <span className={classNames(style.bolder, style.introText)}>{instruction.text}</span>
+    </li>
+)
+
+const Instructions = ({listScreenReaderText, instructions}) => (
+    <div className={classNames(style.thinWrapper, style.introCopy)}>
+      <ul className={style.introBullets} aria-label={listScreenReaderText}>
+      {
+        instructions.map(instruction =>
+          <InstructionPure instruction={instruction} />
+        )
+      }
+      </ul>
+    </div>
+)
+
+class Intro extends Component<Props, State> {
```

Comment:
Minor: this could be a pure component since it only has one render method

Reply:
its using `this.props` for now I think given how long this has gone and given my little experience I am avoiding this change for now